headers/types/common/graphics.h

```diff
@@ -71,4 +72,97 @@ struct animation_control {
 };
 ASSERT_SIZE(struct animation_control, 124);
 
+// Represent a single element to render using the 3D engine
+// This structure is used in two different way. The first one is for planning rendering, that will
+// then call function that will call a function that will then add themm to RENDER_3D for rendering
```

Comment:
> that will then call function that will call a function

Is this intentional because there are two layers of function calls, or is this a typo?